src/utils/client.js

```diff
@@ -81,9 +83,50 @@ async function getAccessToken(url) {
       error(`Exception obtaining an access token from ${server.endpoint}\n${e.message}`);
       throw e;
     }
-  } else {
+  } else if (server) {
     return server.token;
+  } else {
+    error(`Unable to get access token for ${url}`);
+    return;
+  }
+}
+
+/**
+ * Register the backend app with the server and add it to the servers collection
+ *
+ * @param {string} url - the server base url
+ * @returns the server object if successful, otherwise null
+ */
+async function registerServer(url) {
+  if (configUtil.getDynamicClientRegistration() === 'false') return null;
```

Comment:
After the rebase this is now a boolean so 
```suggestion
  if (!configUtil.getDynamicClientRegistration()) return null;
```